Translations/translations_def.js

```diff
@@ -2,367 +2,406 @@ var def =
 		},
 		{
 			"id": "SettingsReset",
 			"maxLen": 8,
-			"maxLen2": 16
+			"maxLen2": 16,
+			"description": "Resets all settings and calibrations to factory defaults. Does NOT erase custom user boot up logo's."
 		},
 		{
 			"id": "VoltageCalibration",
 			"maxLen": 8,
-			"maxLen2": 16
+			"maxLen2": 16,
+			"description": "Enters an adjustment mode where you can gradually adjust the measured voltage to compensate for any unit-to-unit variance in the voltage sense resistors."
 		},
 		{
 			"id": "AdvancedSoldering",
 			"maxLen": 6,
-			"maxLen2": 13
+			"maxLen2": 13,
+			"description": "Should the device show an 'advanced' soldering view. This is a text-based view that shows more information at the cost of no nice graphics."
 		},
 		{
 			"id": "ScrollingSpeed",
 			"maxLen": 6,
-			"maxLen2": 11
+			"maxLen2": 11,
+			"description": "How fast the description text scrolls when hovering on a menu. Faster speeds may induce tearing but allow reading the whole description faster."
 		},
 		{
 			"id": "QCMaxVoltage",
 			"maxLen": 8,
-			"maxLen2": 16
+			"maxLen2": 16,
+			"description": "This adjusts the maximum voltage the QC negotiation will adjust to. Does NOT affect USB-PD. Should be set safely based on the current rating of your power supply."
 		},
 		{
 			"id": "PDNegTimeout",
 			"maxLen": 8,
-			"maxLen2": 16
+			"maxLen2": 16,
+			"description": "How long until firmware stops trying to negotiate for USB-PD and tries QC instead. Longer times may help dodgy/old PD adapters, faster times move onto PD quickly. Units of 100ms. Recommended keeping to small values."
 		},
 
 		{
 			"id": "PowerLimit",
 			"maxLen": 5,
-			"maxLen2": 11
+			"maxLen2": 11,
+			"description": "Allows setting a custom wattage for the iron to aim to keep the AVERAGE power below. The Iron cant control its peak power no matter how you set this. (Except for MHP30 which will regulate nicely to this). If USB-PD is in use, the limit will be set to the lower of this and the supplies advertised wattage."
 		},
 		{
 			"id": "ReverseButtonTempChange",
 			"maxLen": 6,
-			"maxLen2": 16
+			"maxLen2": 16,
+			"description": "Swaps which button increments and decrements on temperature change screens."
 		},
 		{
 			"id": "TempChangeShortStep",
 			"maxLen": 8,
-			"maxLen2": 16
+			"maxLen2": 16,
+			"description": "How much the temperature is increased with a quick press of the buttons."
 		},
 		{
 			"id": "TempChangeLongStep",
 			"maxLen": 6,
-			"maxLen2": 16
+			"maxLen2": 16,
+			"description": "How much the temperature is increased with a long hold of the buttons."
 		},
 		{
 			"id": "PowerPulsePower",
 			"maxLen": 6,
-			"maxLen2": 16
+			"maxLen2": 16,
+			"description": "Enables and sets the wattage of the power pulse. Power pulse causes the iron to briefly turn on the heater to draw power to avoid power banks going to sleep."

```

Comment:
```suggestion
			"description": "Enables and sets the wattage of the power pulse. Power pulse causes the device to briefly turn on the heater to draw power to avoid power banks going to sleep."

```